docs/content/en/getting_started/upgrading.md

```diff
@@ -83,6 +83,24 @@ This requires a one-time rebuild of the Django-Watson search index. Execute the
 
 `./manage.py buildwatson`
 
+**Upgrade instructions for helm chart with postgres enabled**:
```

Comment:
Please write, when this has to be executed. After I have done my normal update routine or does this replace my normal update routine. Mine for example is in a GitLab CI pipeline, that runs after it gets a changed helm chart.

Reply:
I don't know what your update routine does because the migrations should run during the helm update process but if you have one, that should run after the update. The instructions must be executed before the helm upgrade, otherwise you get an error message caused by the statefullset.

Anyway, I added some more lines. Is it more clear now?

Reply:
So I do two helm upgrades, first the one you described here and then the helm upgrade I do normally? Sorry, but I am not really familiar with helm and some other users might not be as well?